cmd/snap/color.go

```diff
@@ -126,24 +128,30 @@ var unicodeDescs = mixinDescs{
 }
 
 type escapes struct {
-	green string
-	bold  string
-	end   string
+	green        string
+	brightYellow string
+	bold         string
+	dim          string
+	end          string
 
-	tick, dash, uparrow string
+	tick, dash, uparrow, star string
 }
 
 var (
 	color = escapes{
-		green: "\033[32m",
-		bold:  "\033[1m",
-		end:   "\033[0m",
+		green:        "\033[32m",
+		brightYellow: "\033[93m",
+		bold:         "\033[1m",
+		dim:          "\033[2m",
```

Comment:
what is dim for? it doesn't seem used

Reply:
I removed it now and added a comment instead in the "mono" part of the code.